posthog/models/feedback/survey.py

```diff
@@ -0,0 +1,66 @@
+from django.db import models
+from posthog.models.utils import UUIDModel
+
+
+class Survey(UUIDModel):
+    class SurveyType(models.TextChoices):
+        POPOVER = "popover", "popover"
+        BUTTON = "button", "button"
```

Comment:
what do button & full screen mean?

Reply:
Button = It's a widget or button that stays on the screen always for users to click and leave feedback on 

Full screen = Survey is a modal that takes up the user's screen instead of a little pop up